scripts/test_names.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Do we want this to be `#!/usr/bin/env bash` instead for slightly more cross-platform compliance?  And then, ideally this script would be marked executable and the tox.ini command wouldn't need to specify `/bin/bash`